lib/consul/migrate/defaults.rb

```diff
@@ -0,0 +1,10 @@
+module Consul
+  module Migrate
+    CLIENT_DEFAULTS = {
+      :bind_client => 'localhost',
+      :port        => 8500
+    }
+  end
+
+  CONFIG_FILE = File.join(Dir.home, '.cmigrate/init.conf')
```

Comment:
Since you're already using `File.join` you can get rid of that hard-coded separator.